src/components/templates/index.ts

```diff
@@ -0,0 +1 @@
+export * from './VideoContentTemplate'
```

Comment:
Let's not use `index.ts` here and import directly, like with the rest of the components